code/api/package.json

```diff
@@ -14,7 +14,7 @@
     "start:prod": "npm run build:prod && npm run start:server",
     "build:prod": "babel src -s -D -d build",
     "start:server": "node build/index.js",
-    "test": "jest"
+    "test": "jest --runInBand --detectOpenHandles"
```

Comment:
`--runInBand` runs tests sequentially. There were issues when running in random order 
`--detectOpenHandles` gets rid of a few errors when running tests